### DISCUSSION TITLE
Is there any plan to support Blend-Shapes (Morph animation) in the near future?

### DISCUSSION THREAD

**Discussion by babefish2008** [skylicht-lab/skylicht-engine#132] [Ideas]
Hi Hung,

Really appreciate to contribute Skylicht-Engine, this engine is very helpful for my app!
I have a look at the feature list and it seems skylicht did't find Blend-shapes(Morph animation) support, right?
Do you have plan to support this feature for glTF or DAE format?

Thank you again!

fish

**Comment by ducphamhong** [comment]
Yes, I have added the blend-shape features example
